[2023-07-01 16:02:23,293] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: API_ETL.conexion_BD manual__2023-07-01T15:57:15.785010+00:00 [queued]>
[2023-07-01 16:02:23,298] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: API_ETL.conexion_BD manual__2023-07-01T15:57:15.785010+00:00 [queued]>
[2023-07-01 16:02:23,298] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-01 16:02:23,299] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-07-01 16:02:23,299] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-01 16:02:23,307] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): conexion_BD> on 2023-07-01 15:57:15.785010+00:00
[2023-07-01 16:02:23,311] {standard_task_runner.py:52} INFO - Started process 315 to run task
[2023-07-01 16:02:23,314] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'API_ETL', 'conexion_BD', 'manual__2023-07-01T15:57:15.785010+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/API_ETL.py', '--cfg-path', '/tmp/tmp_qnuvqex', '--error-file', '/tmp/tmpjoc9qzkv']
[2023-07-01 16:02:23,315] {standard_task_runner.py:80} INFO - Job 22: Subtask conexion_BD
[2023-07-01 16:02:23,366] {task_command.py:371} INFO - Running <TaskInstance: API_ETL.conexion_BD manual__2023-07-01T15:57:15.785010+00:00 [running]> on host c8adaca8488b
[2023-07-01 16:02:23,392] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-07-01 16:02:23,418] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DylanKaplan
AIRFLOW_CTX_DAG_ID=API_ETL
AIRFLOW_CTX_TASK_ID=conexion_BD
AIRFLOW_CTX_EXECUTION_DATE=2023-07-01T15:57:15.785010+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-01T15:57:15.785010+00:00
[2023-07-01 16:02:23,419] {logging_mixin.py:115} INFO - Conectandose a la BD en la fecha: 2023-07-01 15
[2023-07-01 16:02:24,709] {logging_mixin.py:115} INFO - <connection object at 0xffff82583440; dsn: 'user=kaplan46611_coderhouse password=xxx dbname=data-engineer-database host=data-engineer-cluster.cyhh5bfevlmn.us-east-1.redshift.amazonaws.com port=5439', closed: 0>
[2023-07-01 16:02:24,712] {logging_mixin.py:115} INFO - Connected to Redshift successfully!
[2023-07-01 16:02:24,714] {python.py:173} INFO - Done. Returned value was: None
[2023-07-01 16:02:24,746] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=API_ETL, task_id=conexion_BD, execution_date=20230701T155715, start_date=20230701T160223, end_date=20230701T160224
[2023-07-01 16:02:24,790] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-01 16:02:24,817] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
